third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -724,6 +725,37 @@ def AsyncTDMCopyGlobalToLocalOp : TT_AMDGPU_Op<"async_tdm_copy_global_to_local">
     $desc `[` $indices `]` `into` $result `,` $pred
     attr-dict `:` qualified(type($desc)) `->` qualified(type($result))
   }];
+
+  let hasVerifier = 1;
+}
+
+//===----------------------------------------------------------------------===//
+// AsyncTDMCopyLocalToGlobalOp
+//===----------------------------------------------------------------------===//
+
+def AsyncTDMCopyLocalToGlobalOp : TT_AMDGPU_Op<"async_tdm_copy_local_to_global"> {
+  let summary = "Copy data based on descriptor from local memory to global memory asynchronously";
+
+  let description = [{
+    This operation copies data from local memory to global memory
+    asynchronously. This is analogue to tt.store except the data are copied from
+    local memory pointed by `src` instead of a distributed tensor. The copy
+    destination depends on the global memory pointed to by `desc`. This
+    operation does not support shared memory padding or swizzling.
+  }];
+
+  let arguments = (ins
+    Arg<TT_TensorDescType, "", [MemRead<GlobalMemory>, MemWrite<GlobalMemory>]>:$desc,
```

Comment:
We don't have `MemRead<GlobalMemory>` side effect?

Reply:
Yeah I think we don't need it. This originally follows tma as discussed [here](https://github.com/triton-lang/triton/pull/6518/files#r2048646471).

> The TMA engine reads information out of the descriptor. At the same time, it writes to an address stored inside of it.

This should not apply to our current tdm implementation.

Reply:
Right--it's in registers for our side.